### ISSUE TITLE
Error when tying to install as root

### ISSUE THREAD

**Issue by dunatotatos** [amix/vimrc#278]
I wanted to install the vim configuration (awesome version) to be system wide (_ie_ : in /etc/vim).
I did a clone in the associated directory :
```
git clone https://github.com/amix/vimrc.git /etc/vim/vim_runtime
```
Then ran `install_awesome_vimrc.sh`
```
$ sh install_awesome_vimrc.sh 
install_awesome_vimrc.sh: 1: cd: can't cd to /root/.vim_runtime
Installed the Ultimate Vim configuration successfully! Enjoy :-)
```

While I can understand the origin of the error, I think "Installed the Ultimate Vim configuration successfully! Enjoy :-)" shouldn't be displayed if the installation wasn't successful.

I propose two improvements :
* remove this message when the installation is not successful
* allow a system-wide installation in /etc/vim

**Comment by amix** [comment]
Fixed `remove this message when the installation is not successful`.

I am unsure about `/etc/vim`

**Comment by Seliniux777** [comment]
` 'master' of https://github.com/amix/vimrc: (35 commits)
  Updated plugins
  Fixed a conflict between yankstack and surround
  Fixed amix#321
  Fixed amix#325
  Fixed https://github.com/tpope/tpope-vim-abolish
  Updated plugins
  Updated vim plugins
  Speed up download ~4x
  Updated plugins and added vim-abolish
  Allow to install custom plugins
  Updated plugins
  Fixed amix#246
  Fixed amix#263
  Updated screenshots
  Updated the readme (a cleanup)
  Updated the logo
  Fixed amix#278
  Fixed amix#279
  Updated vim plugins
  Removed ack.vim
  ...`
